### PR TITLE
Task/3786 order-details/invoice page redesign

### PR DESCRIPTION
**Pull request by tiwariav**

### CODE REVIEW

**Comment by verysanghee** [comment]
The mobile view needs a little adjustment in icon distance. 
img_4787

**Comment by tiwariav** [comment]
fixed and updated

File: hosting/views.py [hunk 1]
```diff
@@ -644,62 +644,86 @@ class OrdersHostingDetailView(LoginRequiredMixin,
...
                 vm = manager.get_vm(obj.vm_id)
                 context['vm'] = VirtualMachineSerializer(vm).data
             except WrongIdError:
-                messages.error(self.request,
-                               'The VM you are looking for is unavailable at the moment. \
-                                Please contact Data Center Light support.'
-                               )
+                messages.error(
+                    self.request,
+                    'The VM you are looking for is unavailable at the moment. \
```
Comment (pcoder) [inline hosting/views.py]: Not related to this PR, but translating texts shown to the user where possible would be great.

File: hosting/views.py [hunk 2]
```diff
@@ -644,62 +644,86 @@ class OrdersHostingDetailView(LoginRequiredMixin,
...
+                )
                 self.kwargs['error'] = 'WrongIdError'
                 context['error'] = 'WrongIdError'
             except ConnectionRefusedError:
-                messages.error(self.request,
-                               'In order to create a VM, you need to create/upload your SSH KEY first.'
-                               )
+                messages.error(
+                    self.request,
+                    'In order to create a VM, you need to create/upload your SSH KEY first.'
```
Comment (pcoder) [inline hosting/views.py]: Here too.

**Pull request merged**